### PR TITLE
CART-50 added ui and logical part for admin employees

### PR DESCRIPTION
**Pull request by RomanFedytskyi**

### CODE REVIEW

File: dev/publicDev/admin.employees/style.scss [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  }
+
+  .employees-content {
+    margin-top: 40px;
+    width: 100%;
+    padding: 0 70px;
+    .name, .email {
+      display: inline-block;
+      width: 35%;
+      text-overflow: ellipsis;
```
Comment (dmitrij-borchuk) [inline dev/publicDev/admin.employees/style.scss]: Use class ".ellipsis"

File: dev/routing/users.js [hunk 2]
```diff
@@ -42,4 +42,76 @@ module.exports = function (server) {
...
+  });
+
+  server.route({
+    method: 'post',
+    path: '/api/user-block/{id}',
+    config: {
+      auth: 'simple',
+      plugins: {
+        hapiRouteAcl: {
+          permissions: ['users:read']
```
Comment (dmitrij-borchuk) [inline dev/routing/users.js]: Should be "users:write"

File: dev/routing/users.js [hunk 3]
```diff
@@ -42,4 +42,76 @@ module.exports = function (server) {
...
+  });
+
+  server.route({
+    method: 'PUT',
+    path: '/api/users/{id}',
+    config: {
+      auth: 'simple',
+      plugins: {
+        hapiRouteAcl: {
+          permissions: ['users:read']
```
Comment (dmitrij-borchuk) [inline dev/routing/users.js]: Should be "users:write"

File: dev/routing/users.js [hunk 4]
```diff
@@ -42,4 +42,76 @@ module.exports = function (server) {
...
+  });
+
+  server.route({
+    method: 'post',
+    path: '/api/user-unblock/{id}',
+    config: {
+      auth: 'simple',
+      plugins: {
+        hapiRouteAcl: {
+          permissions: ['users:read']
```
Comment (dmitrij-borchuk) [inline dev/routing/users.js]: Should be "users:write"

**Review by dmitrij-borchuk — CHANGES_REQUESTED** [review]

File: dev/routing/users.js (hunk 4 above)
Comment (RomanFedytskyi) [reply dev/routing/users.js]: fixed

File: dev/routing/users.js (hunk 2 above)
Comment (RomanFedytskyi) [reply dev/routing/users.js]: fixed

File: dev/routing/users.js (hunk 3 above)
Comment (RomanFedytskyi) [reply dev/routing/users.js]: fixed

File: dev/publicDev/admin.employees/style.scss (hunk 1 above)
Comment (RomanFedytskyi) [reply dev/publicDev/admin.employees/style.scss]: fixed

**Pull request merged**